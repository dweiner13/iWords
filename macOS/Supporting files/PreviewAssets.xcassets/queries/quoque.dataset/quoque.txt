que                  TACKON                             
PACKON w/qui => whoever it be; whatever; each, each one; everyone, everything;
qu.o                 PRON   1 0 ABL S M                 
qu.o                 PRON   1 0 ABL S N                 
qu  PACK  
(w/-que) each, each one; every, everybody, everything (more than 2); whatever;
(w/-que) any; each;
quoque               ADV    POS                         
quoque  ADV  
likewise/besides/also/too; not only; even/actually; (after word emphasized);
que                  TACKON                             
-que = and (enclitic, translated before attached word); completes plerus/uter;
qu.o                 PRON   1 0 ABL S M                 
qu.o                 PRON   1 0 ABL S N                 
who; that; which, what; of which kind/degree; person/thing/time/point that;
who/what/which?, what/which one/man/person/thing? what kind/type of?;
who/whatever, everyone who, all that, anything that;
any; anyone/anything, any such; unspecified some; (after si/sin/sive/ne);
who?, which?, what?; what kind of?;
quo                  ADV    POS                         
quo  ADV    lesser
where, to what place; to what purpose; for which reason, therefore;
quo                  CONJ                               
quo  CONJ    lesser
whither, in what place, where;
que                  TACKON                             
PACKON w/qui => whoever it be; whatever; each, each one; everyone, everything;
qu.o                 PRON   1 0 ABL S M                 
qu.o                 PRON   1 0 ABL S N                 
qu  PACK  
(w/-que) each, each one; every, everybody, everything (more than 2); whatever;
(w/-que) any; each;

