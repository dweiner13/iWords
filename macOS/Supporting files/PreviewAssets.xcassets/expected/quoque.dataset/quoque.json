[{"definition":{"_0":{"words":[{"meaning":"PACKON w\/qui => whoever it be; whatever; each, each one; everyone, everything;"}],"truncated":false,"possibilities":[{"tackon":{"_0":"que"}}]}}},{"definition":{"_0":{"words":[{"meaning":"qu  PACK  \n(w\/-que) each, each one; every, everybody, everything (more than 2); whatever;\n(w\/-que) any; each;"}],"truncated":false,"possibilities":[{"pronoun":{"_0":{"declension":1,"case":"ABL","number":"S","gender":"M","variety":0,"text":"qu.o"}}},{"pronoun":{"_0":{"declension":1,"case":"ABL","number":"S","gender":"N","variety":0,"text":"qu.o"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"likewise\/besides\/also\/too; not only; even\/actually; (after word emphasized);","expansion":{"adv":{"_0":"quoque","_1":[]}}}],"truncated":false,"possibilities":[{"adverb":{"_0":{"degree":"POS","text":"quoque"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"-que = and (enclitic, translated before attached word); completes plerus\/uter;"}],"truncated":false,"possibilities":[{"tackon":{"_0":"que"}}]}}},{"definition":{"_0":{"words":[{"meaning":"who; that; which, what; of which kind\/degree; person\/thing\/time\/point that;\nwho\/what\/which?, what\/which one\/man\/person\/thing? what kind\/type of?;\nwho\/whatever, everyone who, all that, anything that;\nany; anyone\/anything, any such; unspecified some; (after si\/sin\/sive\/ne);\nwho?, which?, what?; what kind of?;"}],"truncated":false,"possibilities":[{"pronoun":{"_0":{"declension":1,"case":"ABL","number":"S","gender":"M","variety":0,"text":"qu.o"}}},{"pronoun":{"_0":{"declension":1,"case":"ABL","number":"S","gender":"N","variety":0,"text":"qu.o"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"where, to what place; to what purpose; for which reason, therefore;","expansion":{"adv":{"_0":"quo","_1":["lesser"]}}}],"truncated":false,"possibilities":[{"adverb":{"_0":{"degree":"POS","text":"quo"}}}]}}},{"definition":{"_0":{"words":[{"meaning":"whither, in what place, where;","expansion":{"conj":{"_0":"quo","_1":["lesser"]}}}],"truncated":false,"possibilities":[{"conjunction":{"_0":"quo"}}]}}},{"definition":{"_0":{"words":[{"meaning":"PACKON w\/qui => whoever it be; whatever; each, each one; everyone, everything;"}],"truncated":false,"possibilities":[{"tackon":{"_0":"que"}}]}}},{"definition":{"_0":{"words":[{"meaning":"qu  PACK  \n(w\/-que) each, each one; every, everybody, everything (more than 2); whatever;\n(w\/-que) any; each;"}],"truncated":false,"possibilities":[{"pronoun":{"_0":{"declension":1,"case":"ABL","number":"S","gender":"M","variety":0,"text":"qu.o"}}},{"pronoun":{"_0":{"declension":1,"case":"ABL","number":"S","gender":"N","variety":0,"text":"qu.o"}}}]}}}]